### ISSUE TITLE
do not add duplicate remote with the name name

### ISSUE THREAD

**Issue by vincenzopalazzo** [coffee-tools/coffee#108]
We shuold not allow add a remote with the same name of another one

```
➜  coffee git:(macros/termianl-ui) ✗ cargo run --bin coffee -- --network testnet remote list

    Finished dev [unoptimized + debuginfo] target(s) in 0.06s
     Running `target/debug/coffee --network testnet remote list`
● List of repositories
╭─────────────────────────────────────────────────────────────────────────────╮
│ ●   Repository Alias   URL                                       N. Plugins │
├─────────────────────────────────────────────────────────────────────────────┤
│ ●   folgore-git        https://github.com/coffee-tools/folgore   1          │
│ ●   lightningd         https://github.com/lightningd/plugins     27         │
╰─────────────────────────────────────────────────────────────────────────────╯
➜  coffee git:(macros/termianl-ui) ✗ cargo run --bin coffee -- --network testnet remote add lightningd https://github.com/lightningd/plugins.git

    Finished dev [unoptimized + debuginfo] target(s) in 0.06s
     Running `target/debug/coffee --network testnet remote add lightningd 'https://github.com/lightningd/plugins.git'`
✓ Remote added!
➜  coffee git:(macros/termianl-ui) ✗ cargo run --bin coffee -- --network testnet remote list                                                    

    Finished dev [unoptimized + debuginfo] target(s) in 0.06s
     Running `target/debug/coffee --network testnet remote list`
● List of repositories
╭─────────────────────────────────────────────────────────────────────────────╮
│ ●   Repository Alias   URL                                       N. Plugins │
├─────────────────────────────────────────────────────────────────────────────┤
│ ●   folgore-git        https://github.com/coffee-tools/folgore   1          │
│ ●   lightningd         https://github.com/lightningd/plugins     27         │
│ ●   lightningd         https://github.com/lightningd/plugins     27         │
╰─────────────────────────────────────────────────────────────────────────────╯
➜  coffee git:(macros/termianl-ui) ✗ 

```

**Comment by vincenzopalazzo** [comment]
it is better see if the code allow this or when the code allow this

**Comment by tareknaser360** [comment]
solved by  #115
